README.rst

```diff
@@ -69,10 +69,16 @@ Example usage::
 generates a file called ``migrations/20151217170514_add_id_to_users.py``
 with content::
 
+    # Uncomment should_run if this is a repeat migration
+    # def should_run(cursor):
```

Comment:
In which case would a migration run more than once? I thought that the DB would just have a table with all the migrations that successfully ran, no?